src/csharp/Sequences.cs

```diff
@@ -0,0 +1,70 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
```

Comment:
All classes and methods should be documented

Reply:
Would it be ok to get to the API documentation in another PR to avoid delaying other tasks dependent on this PR?